qtconsole/qt.py

```diff
@@ -1,23 +1,5 @@
 """ A Qt API selector that can be used to switch between PyQt4/5 and PySide.
```

Comment:
I think this submodule should be removed and `qtpy` used directly wherever is needed.